### ISSUE TITLE
Let escape fields with dots

### ISSUE THREAD

**Issue by algorys** [apidoc/apidoc#774]
I know that dots are used for `Array` and `Object`, but in case we receive (or must give) field with dot inside, we can't have something like:

```
* @apiSuccess {String} test.hello A test for hello
* @apiSuccess {String} test.mail  A mail for hello
```
Because apiDoc convert to:

```
hello    String    A test for hello
mail     String    A mail for hello
```

Can we have something to escape this ? Like the following:

```
* @apiSuccess {String} "test.hello" A test for hello
* @apiSuccess {String} "test.mail"  A mail for hello
# or
* @apiSuccess (_items) {String} _items."test.hello" A test for hello
* @apiSuccess (_items) {String} _items."test.mail" A test for hello
```

That will render to:

```
test.hello    String    A test for hello
test.mail     String    A mail for hello
# Or
_items
    test.hello    String    A test for hello
    test.mail     String    A mail for hello
```
I propose double quotes, but maybe another character is better ?

**Comment by arthurtalkgoal** [comment]
Agree with @algorys , I just met the same problem!

**Comment by tomislav13** [comment]
This is what I need

**Comment by gazdagergo** [comment]
I have similar issue with query params:
```
@apiParam {String} city.street
``` 
is being rendered as 
```

![image](https://user-images.githubusercontent.com/8988279/79118201-7bda8280-7d8d-11ea-8a40-b6f5df293dd3.png)

**Comment by NicolasCARPi** [comment]
Fixed in #1063
